tasks/gitlab_helpers.py

```diff
@@ -81,3 +82,45 @@ def create_gitlab_annotations_report(ci_job_id: str, ci_job_name: str):
             },
         ]
     }
+
+
+def print_gitlab_object(
+    get_object: callable, ctx, id, repo='DataDog/datadog-agent', jq: str | None = None, jq_colors=True
+):
+    """
+    Print one or more Gitlab objects in JSON and potentially query them with jq
+    """
+    repo = get_gitlab_repo(repo)
+    ids = [i for i in id.split(",") if i]
```

Comment:
nit - you could also use [`filter`](https://docs.python.org/3/library/functions.html#filter)

Reply:
Yep, I think that Ruff replaces filter by this as it is more explicit